### PR TITLE
Elasticsearch 1.2.4 and 1.3.9

### PR DESCRIPTION
**Pull request by stringbean**
These are in `elasticsearch12` and `elasticsearch13` respectively.

### CODE REVIEW

**Comment by DomT4** [comment]
Hi! If this is still useful please address any comments ` brew audit --strict ` has locally, Thanks!

File: elasticsearch12.rb [hunk 1]
```diff
@@ -0,0 +1,122 @@
+require "formula"
+
+class Elasticsearch12 < Formula
+  homepage "http://www.elasticsearch.org"
+  url "https://download.elasticsearch.org/elasticsearch/elasticsearch/elasticsearch-1.2.4.tar.gz"
+  sha1 "c8bbe1f1975ffb6774744fadd0abb78616e96904"
+
+  depends_on :java => "1.7"
```
Comment (DomT4) [inline elasticsearch12.rb]: ` depends_on :java => "1.7+" ` (Same for the formula below as well)

File: elasticsearch12.rb [hunk 2]
```diff
@@ -0,0 +1,122 @@
...
+
+  head do
```
Comment (DomT4) [inline elasticsearch12.rb]: :scissors: the head blocks. Versions formulae can't have ` head ` blocks.

**Comment by stringbean** [comment]
@DomT4 I've made the changes you suggested and fixed the audit complaints.

**Comment by DomT4** [comment]
Merged! Thanks for your contribution to Homebrew, We appreciate it! For future reference new formulae should be split into multiple commits when they don't depend on each other, and the commit reference should be ` formulaname version (new formula) `, Thanks!

**Pull request closed without merging**